frontend/src/concepts/modelCatalog/useModelCatalogSources.ts

```diff
@@ -27,22 +28,28 @@ export const useModelCatalogSources = (): FetchState<State> => {
       throw new NotReadyError('Model catalog feature is not enabled');
     }
 
-    try {
-      const configMap = await getConfigMap(dashboardNamespace, MODEL_CATALOG_SOURCES_CONFIGMAP);
-      if (!configMap.data || !configMap.data.modelCatalogSources) {
-        return [];
+    const [successes, fails] = await allSettledPromises([
+      getConfigMap(dashboardNamespace, MODEL_CATALOG_SOURCES_CONFIGMAP),
+      getConfigMap(dashboardNamespace, 'model-catalog-unmanaged-sources'),
```

Comment:
`'model-catalog-unmanaged-sources'` should be a constant like `MODEL_CATALOG_SOURCES_CONFIGMAP` is